samples/Azure.Management.Storage/Generated/BlobContainer.cs

```diff
@@ -38,44 +39,22 @@ protected BlobContainer()
         }
 
         /// <summary> Initializes a new instance of the <see cref = "BlobContainer"/> class. </summary>
-        /// <param name="options"> The client parameters to use in these operations. </param>
+        /// <param name="armClient"> The client parameters to use in these operations. </param>
         /// <param name="data"> The resource that is the target of operations. </param>
-        internal BlobContainer(ArmResource options, BlobContainerData data) : base(options, data.Id)
+        internal BlobContainer(ArmClient armClient, BlobContainerData data) : this(armClient, data.Id)
```

Comment:
Can we remove `ArmResource(ArmResource parent, ..)` ctor now?

Reply:
yes after this goes in I will be removing a bunch of stuff inside this pr https://github.com/Azure/azure-sdk-for-net/pull/26452